### DISCUSSION TITLE
My Github page failed to deploy fully

### DISCUSSION THREAD

**Discussion by wqw547243068** [community/community#31491] [Other Feature Feedback, Questions, & Ideas] [answered]
## Background

- I have deployed a [Github Page]() for many years
- Today, I extracted some paragraphs from one post, and merged into a new post
  - The original page: [dm-note](https://github.com/wqw547243068/wqw547243068.github.io/blob/master/_posts/2013-07-31-dm-note.md)
  - The newly created page: [regression](https://github.com/wqw547243068/wqw547243068.github.io/blob/master/_posts/2022-08-31-regression.md)

## Question

- Both pages are <font color='red'> deployed correctly,</font> the [old one](https://wqw547243068.github.io/2013/07/31/dm-note) is displayed normally, but the newly [created page](https://wqw547243068.github.io/2022/08/31/regression) is <font color='red'>never displayed</font>
  - <img width="1243" alt="image" src="https://user-images.githubusercontent.com/3257257/187685614-5ce9ab5d-50e1-44ec-8e94-2ff8c918e50f.png">
- I don't know why ...

**Comment by jfullstackdev** [comment]
your last commit was 35 minutes ago as of this posting, is that the latest one ? is it now reflected on GitHub Pages ? if not, try logging in in Incognito mode

![image](https://user-images.githubusercontent.com/47092464/187700727-863ce51c-e9ae-41ab-b36d-7e2239f32d5d.png)

**Reply by wqw547243068** [reply]
Problem solved, thank you for answering! The post downstairs indicates the true reason: 
> date of the post is ahead of deployment time

**Comment by airtower-luna** [comment, marked as answer]
The [build log](https://github.com/wqw547243068/wqw547243068.github.io/runs/8114786643?check_suite_focus=true#step:4:187) tells you why:

> Skipping: _posts/2022-08-31-regression.md has a future date

Indeed the date listed in the file metadata is over 2h from now (2022-08-31 14:40 UTC):

```yaml
date:   2022-08-31 17:02:00
```

Adjust the date to one that's in the past, or redeploy after the time has passed. :slightly_smiling_face:

**Reply by wqw547243068** [reply]
Got it, this post displays now, you are right, the date of post is ahead of deployment time 
Thanks a lot !
